cypress.config.js

```diff
@@ -0,0 +1,28 @@
+const { defineConfig } = require('cypress');
+
+module.exports = defineConfig({
+  viewportWidth: 1920,
+  viewportHeight: 1080,
+  video: false,
+  defaultCommandTimeout: 101000,
+  pageLoadTimeout: 120000,
+  env: {
+    OKAPI_HOST: 'https://folio-testing-cypress-okapi.ci.folio.org',
+    OKAPI_TENANT: 'diku',
+    diku_login: 'diku_admin',
+    diku_password: 'admin',
+    is_kiwi_release: false,
+    downloadTimeout: 1000,
+    allure: 'true',
+    grepFilterSpecs: true,
+    grepOmitFiltered: true,
+  },
+  e2e: {
+    // We've imported your old cypress plugins here.
+    // You may want to clean this up later by importing these.
+    setupNodeEvents(on, config) {
+      return require('./cypress/plugins/index.js')(on, config);
+    },
+    baseUrl: 'https://folio-testing-cypress-diku.ci.folio.org',
```

Comment:
Please move baseUrl to the fourth code line for convenience

Reply:
BaseUrl lies inside e2e object, I can not move it on 4 line